external-docs/docs/guides/shape-data.md

```diff
@@ -34,208 +34,3 @@ _Data set representing the number of 'Arthur's Bakes' stores in UK cities from 2
     * the `Status` _attribute_ column contains information describing the state of the observed value itself.
     * Note that _attributes_ should only describe the observed value and must not identify any subset of the population.  
 
-## Standard shape
```

Comment:
Can we have short descriptions and basic examples of the standard and pivoted shapes here with links to the relevant docs for more info?

Reply:
Done